x/move/keeper/balancer.go

```diff
@@ -206,7 +206,7 @@ func (k BalancerKeeper) poolMetadata(ctx context.Context, metadataLP vmtypes.Acc
 		return nil, err
 	}
 
-	metadataA, _, err := k.Keeper.MoveBankKeeper().Balance(ctx, storeA)
+	metadata, _, err := k.Keeper.MoveBankKeeper().Balance(ctx, storeA)
```

Comment:
should keep metadataA to indicate it is metadata of coin A

Reply:
reverted